### PR TITLE
iterative search slides

### PR DESCRIPTION
**Pull request by topepo**

### CODE REVIEW

File: slides/advanced-03-iterative.qmd [hunk 1]
```diff
@@ -0,0 +1,658 @@
...
+```
+
+## The Final Fit
+
+We can use individual functions: 
+
+```r
+final_fit <- final_wflow %>% fit(data = hotel_tr)
+
+# the predict() or augment() then compute metrics
```
Comment (hfrick) [inline slides/advanced-03-iterative.qmd:587]:
```suggestion
# then predict() or augment() 
# then compute metrics
```

Are you not showing the corresponding code on purpose?

**Review by hfrick — COMMENTED** [review]
Some small comments on the slide content and two "higher level" comments not content-related:

- There is a bit of a hodgepodge going on with file and folder names re `advanced-03-iterative` vs `advanced-03-squential`.
- The "Your turn" slides aren't styled in the same way as the initial ones.

File: slides/advanced-03-iterative.qmd [hunk 2]
```diff
@@ -0,0 +1,658 @@
...
+Exploration focuses on the variance, exploitation is about the mean. 
+
+## Acquisition Functions
+
+We'll use an acquisition function to select a new candidate.
+
+The most popular method appears to be _expected improvement_ ([EI](https://arxiv.org/pdf/1911.12809.pdf)) above the current best results. 
+ 
+  - Zero at existing data points. 
+  - The _expected_ improvement is integrated overall all possible improvement ("expected" in the probability sense).
```
Comment (simonpcouch) [inline slides/advanced-03-iterative.qmd:311]:
```suggestion
  - The _expected_ improvement is integrated over all possible improvement ("expected" in the probability sense). 
```

File: slides/advanced-03-iterative.qmd [hunk 3]
```diff
@@ -0,0 +1,658 @@
...
+```{r}
+#| label: gp-anime
+#| echo: false
+#| out-width: 50%
+#| fig-align: center
+#| fig-width: 6
+#| fig-height: 4.25
+
+knitr::include_graphics("animations/anime_gp.gif")
+```
```
Comment (simonpcouch) [inline slides/advanced-03-iterative.qmd:401]: I don't see any content on this or the next slide when I look at the rendered html. Do others have this issue?

File: slides/advanced-03-iterative.qmd [hunk 4]
```diff
@@ -0,0 +1,658 @@
...
+#| out-width: 50%
+#| fig-align: center
+#| fig-width: 6
+#| fig-height: 4.25
+
+autoplot(lgbm_bayes_res, metric = "mae", type = "performance")
+```
+
+
+## ENCHANCE
```
Comment (simonpcouch) [inline slides/advanced-03-iterative.qmd:513]: Googled this and it does seem to be a thing, but confirming that this wasn't supposed to be "enhance"?

**Review by simonpcouch — APPROVED** [review]
This is in a really good spot!🐬

File: slides/advanced-03-iterative.qmd (hunk 4 above)
Comment (topepo) [reply slides/advanced-03-iterative.qmd:513]: typo

File: slides/advanced-03-iterative.qmd (hunk 3 above)
Comment (topepo) [reply slides/advanced-03-iterative.qmd:401]: On a separate computer, I fetched the PR and compiled it. It is there for me. Is the EI gif on the next slide working?

Comment (simonpcouch) [reply slides/advanced-03-iterative.qmd:416]: So strange--maybe another oddity from my differing revealjs install. Will leave this for after NYR👍

**Pull request closed without merging**